fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -177,15 +177,23 @@ public String toString() {
     }
 
     public static class AsyncRefreshContext {
+        // Olap base table refreshed meta infos
         // base table id -> (partition name -> partition info (id, version))
         // partition id maybe changed after insert overwrite, so use partition name as key.
         // partition id which in BasePartitionInfo can be used to check partition is changed
         @SerializedName("baseTableVisibleVersionMap")
         private final Map<Long, Map<String, BasePartitionInfo>> baseTableVisibleVersionMap;
 
+        // External base table refreshed meta infos
         @SerializedName("baseTableInfoVisibleVersionMap")
         private final Map<BaseTableInfo, Map<String, BasePartitionInfo>> baseTableInfoVisibleVersionMap;
 
+        // Materialized view partition is updated/added associated with ref-base-table. This meta
+        // is kept to track materialized view's partition change and update associated ref-base-table
+        // at the same time.
+        @SerializedName("mvPartitionNameRefBaseTablePartitionMap")
+        private final Map<String, Set<String>> mvPartitionNameRefBaseTablePartitionMap;
```

Comment:
what's the meaning of its key and value

Reply:
Key    : materialized view's partition
Value :  associated ref-base-table partitions